### ISSUE TITLE
Top bar is clipped on non-responsive pages on responsive sites, e.g. profiles on MSE

### ISSUE THREAD

**Issue by cellio** [vyznev/soup#41]
SE recently started deploying a new responsive design, but not all pages are responsive yet.  Q&A is, but profiles and reviews aren't.

I recently noticed a problem with the top bar on those non-responsive pages and filed a bug report, but it turns out to be coming from something SOUP is doing.  (I disabled user scripts one at a time until I found the one that introduced the change.)  That bug report, with screen shots, is here: https://meta.stackexchange.com/q/311489/162102 .  Basically, in a smaller window size (mine is 1036px wide), the right side of the top bar is clipped off only on these pages.  On sites that don't have the responsive design yet (almost all of them) and responsive pages on SO and Meta.SE, the problem does not occur.

Eventually they're going to make everything responsive and presumably the problem will go away, but it's going to take them weeks to months to do that.  Is there anything that can be done in the meantime?  I have no idea how large or small an investigation/change this would be.

I really appreciate all the work you've put into SOUP!  I'm sorry to come here asking for a fix instead of working it out and offering you a PR.

In case it matters, I'm using the userscript version, not the add-on.

**Comment by vyznev** [comment]
Thanks for reporting this. The problematic fix was [mso345590](https://meta.stackoverflow.com/q/345590), which a bit ironically was meant to fix exactly the same UI issue that you observed, i.e. the right side of the top bar getting clipped in narrow browser windows.

Unfortunately, the implementation of this fix in SOUP was not fully compatible with the new responsive top bar design. I had already disabled this fix for responsive pages, but I hadn't realized that the new top bar design was actually responsive even when the page content wasn't. So the fix was basically doing what it was meant to do (i.e. forcing the top bar to be at least as wide as the content area, so that it could be scrolled horizontally to bring all the menus into view) even though that was no longer needed or useful.

Anyway, I've removed the problematic fix entirely in SOUP v1.52.3 (devel v1.53.6).

As a side effect of the removal, non-responsive pages on sites with a non-sticky top bar may now show an ugly white box to the right of the top bar if the page is scrolled horizontally. But that cosmetic issue happens with or without SOUP, anyway, and I'm not going to even try fixing it until the responsive design stabilizes. Hopefully, it will get fixed by SE before that.
